src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/PersistenceServiceTest.java

```diff
@@ -39,22 +36,14 @@
 import javax.persistence.criteria.Predicate;
 import javax.persistence.criteria.Root;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Similar to above : We should avoid using * while import packages..It may import unnecessary classes as well.

Reply:
Done